docs/contributing-analyzer.md

```diff
@@ -15,16 +15,17 @@ In general, it is best to run commands from the Visual Studio Developer Command
 1. [Visual Studio 2019](https://visualstudio.microsoft.com/vs/)
 1. Ensure to install Workloads:
     - ASP.NET and web development
-    - .NET desktop development (C#, VB and F# with .NET Core and .NET Framework)
+    - .NET desktop development
     - Visual Studio extension development
+    - .NET Core cross-platform development
 1. Ensure to install Individual components:
     - .NET Core 3.1 SDK
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
     - .NET 3.5 SDK (SP1)
+    - .NET Compiler Platform SDK
 1. The following environment variables must be set:
     - **JAVA_HOME**
-    - **MAVEN_HOME** / **M2_HOME**
```

Comment:
Unused. I don't have them on my machine (never had) and I haven't found them anywhere in the project.

Reply:
I probably added everything I had locally, and maybe this was needed in other projects